book/src/specs/kimchi.md

```diff
@@ -1720,28 +1720,40 @@ You can find these operations under the [proof creation](#proof-creation) and [p
 A proof consists of the following data structures:
 
 ```rs
-/// Evaluations of lookup polynomials
+/// Evaluations of a polynomial at 2 points
 #[serde_as]
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Copy, Clone, Serialize, Deserialize, Default)]
+#[cfg_attr(
+    feature = "ocaml_types",
+    derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Struct)
+)]
 #[serde(bound(
-    serialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::SerializeAs<Field>",
-    deserialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::DeserializeAs<'de, Field>"
+    serialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::SerializeAs<Evals>",
+    deserialize = "Vec<o1_utils::serialization::SerdeAs>: serde_with::DeserializeAs<'de, Evals>"
 ))]
-pub struct LookupEvaluations<Field> {
+pub struct PointEvaluations<Evals> {
+    /// Evaluation at the challenge point zeta.
+    #[serde_as(as = "Vec<o1_utils::serialization::SerdeAs>")]
+    pub zeta: Evals,
+    /// Evaluation at `zeta . omega`, the product of the challenge point and the group generator.
+    #[serde_as(as = "Vec<o1_utils::serialization::SerdeAs>")]
```

Comment:
I don't understand the change of `Vec<E>` to `E`, what is the rationale here? This makes the code harder to follow imo